test/syncenginetestutils.cpp

```diff
@@ -188,6 +188,13 @@ void FileInfo::setModTime(const QString &relativePath, const QDateTime &modTime)
     file->lastModified = modTime;
 }
 
+void FileInfo::setModTimeKeepEtag(const QString &relativePath, const QDateTime &modTime)
+{
+    FileInfo *file = find(relativePath);
+    Q_ASSERT(file);
+    file->lastModified = modTime;
```

Comment:
@mgallien don't you want to check the pointer before resolution? Such tat we don't have a crash in runtime. And maybe a log if it is `null`.

Reply:
that was what I wanted to do with `Q_ASSERT(file);` as we are in the automated test code and I expect people to build it in debug mode